fastlane/lib/fastlane/plugins/templates/action.rb.erb

```diff
@@ -0,0 +1,35 @@
+module Fastlane
+  module Actions
+    class <%= plugin_name.capitalize %>Action < Action
+      def self.run(params)
+        UI.message("The <%= plugin_name %> plugin is working!")
+      end
+
+      def self.description
+        %q{<%= summary %>}
```

Comment:
Hm, trying to manually escape user input is usually a bad idea, as we have seen in other places 😅 But if you feel strongly, we can try it!